gdx/src/com/badlogic/gdx/graphics/g2d/Animation.java

```diff
@@ -80,6 +80,7 @@ public Animation (float frameDuration, T... keyFrames) {
 
 	/** Returns a frame based on the so called state time. This is the amount of seconds an object has spent in the state this
 	 * Animation instance represents, e.g. running, jumping and so on. The mode specifies whether the animation is looping or not.
+	 * But if the animation is set to LOOP_PINGPONG and LOOP_RANDOM mode, non-loop param takes no effect.
```

Comment:
I have some minor comment nitpicks. It should be:

`LOOP_PINGPONG or LOOP_RANDOM`

and the verb in `param takes no effect` should be changed to `param has no effect` .

It's clear enough already that it can be merged now, but if you want to polish it before merging that would be nice.